app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -91,11 +150,61 @@ class SearchActivity : AppCompatActivity() {
         searchEdit.setText(editValue)
     }
 
+
+    private fun enterSearch() {
+        if (searchEdit.text.isNotEmpty()) {
+            iTunesService.search(searchEdit.text.toString()).enqueue(object :
+                Callback<TracksResponse> {
+
+                @SuppressLint("NotifyDataSetChanged")
+                override fun onResponse(
+                    call: Call<TracksResponse>,
+                    response: Response<TracksResponse>
+                ) {
+                    if (response.code() == 200) {
```

Comment:
```suggestion
                    if (response.isSuccessfull) {
```
Лучше написать так, чтобы не хардкодить коды http запросов